int32 x
int32 y
float32 depth
float32 fx
float32 fy
float32 px
float32 py
---
float32 X
float32 Y
float32 Z
